spec/pivo_flow/pivotal_spec.rb

```diff
@@ -6,34 +6,37 @@
 
   before do
     stub_git_config
-    PivoFlow::Pivotal.any_instance.stub(:puts)
+
+    allow_any_instance_of(PivoFlow::Pivotal).to receive(:puts)
+
     stub_base_methods(PivoFlow::Pivotal)
   end
 
   describe "raises exception" do
     it "when project id is incorrect" do
       VCR.use_cassette(:pivotal_fetch_project_not_found) do
-        expect{pivotal.fetch_stories}.to raise_error(PivoFlow::Errors::UnsuccessfulPivotalConnection)
+        expect{ pivotal.fetch_stories }.to \
+          raise_error(PivoFlow::Errors::UnsuccessfulPivotalConnection)
       end
     end
 
     it "when api-token is incorrect (unauthorized)" do
       VCR.use_cassette(:pivotal_fetch_project_unauthorized) do
-        expect{pivotal.fetch_stories}.to raise_error(PivoFlow::Errors::UnsuccessfulPivotalConnection)
+        expect{ pivotal.fetch_stories }.to \
```

Comment:
can we omit the `\` sign and go with

```ruby
expect { pivotal.fetch_stories }
  .to raise_error(PivoFlow::Errors::UnsuccessfulPivotalConnection)
```

?

Reply:
yep